README.md

```diff
@@ -2143,6 +2143,8 @@ mongoimport -u admin -p <password> -d <database_name> -c settings mongo_settings
 mongoimport -u admin -p <password> -d <database_name> --jsonArray -c roles mongo_roles.json
 ```
 
+If this command don't work for you, try yo check [https://docs.mongodb.com/v4.2/reference/program/mongoimport/](https://docs.mongodb.com/v4.2/reference/program/mongoimport/)
```

Comment:
**don't** -> **doesn't**
**yo** -> **to**

Reply:
Ty, done.